scripts/choco-install.ps1

```diff
@@ -0,0 +1,39 @@
+$nodejs = "nodejs.install"
```

Comment:
Give the `scripts/choco-install.ps1` executable permission. Like `chmod u+x`.

Reply:
Done